pkg/scheduler/plugins/elasticquota/core/quota_info.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"sync"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	v1 "k8s.io/api/core/v1"
+	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+)
+
+type QuotaCalculateInfo struct {
+	// The semantics of "max" is the quota group's upper limit of resources.
+	Max v1.ResourceList `json:"max"`
+	// The semantics of "min" is the quota group's guaranteed resources, if quota group's "request" less than or
+	// equal to "min", the quota group can obtain equivalent resources to the "request"
+	OriginalMin v1.ResourceList `json:"original_min"`
+	// If Child's sumMin is larger than totalResource, the value of OriginalMin should be scaled in equal proportion
+	//to ensure the correctness and fairness of min
+	AutoScaleMin v1.ResourceList `json:"auto_scale_min"`
+	// All assigned pods used
+	Used v1.ResourceList `json:"used"`
+	// All pods request
+	Request v1.ResourceList `json:"request"`
+	// SharedWeight determines the ability of quota groups to compete for shared resources
+	SharedWeight v1.ResourceList `json:"shared_weight"`
+	// Runtime is the current actual resource that can be used by the quota group
+	Runtime v1.ResourceList `json:"runtime"`
+}
+
+type QuotaInfo struct {
+	// QuotaName
+	Name QuotaNameType `json:"name"`
```

Comment:
Do we really need the `QuotaNameType` type?  Wouldn't it be okay to just use `string`?